_posts/en/newsletters/2023-03-08-newsletter.md

```diff
@@ -0,0 +1,147 @@
+---
+title: 'Bitcoin Optech Newsletter #241'
+permalink: /en/newsletters/2023/03/08/
+name: 2023-03-08-newsletter
+slug: 2023-03-08-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Alternative design for OP_VAULT:** Greg Sanders [posted][sanders
+  vault] to the Bitcoin-Dev mailing list an alternative design for
+  providing the features of the `OP_VAULT`/`OP_UNVAULT` proposal (see
+  [Newsletter #234][news234 vault]).  His alternative would add three
+  opcodes instead of two.  To provide an example:
+
+    - *Alice deposits funds in a vault* by paying a [taproot][topic
+      taproot] tree which contains at least two [tapscripts][topic
+      tapscript], one with her unvault tapscript and one with her freeze
+      tapscript, e.g. `tr(key,{trigger,freeze})`.
+
+      - The *trigger tapscript* contains her less-trusted authorization
+        conditions (such as requiring a signature from her hot wallet)
+        and an `OP_TRIGGER_FORWARD` opcode.  At the time she creates
+        this tapscript, she provides the opcode a *spend delay*
+        parameter, e.g. a relative timelock of 1,000 blocks (about 1
+        week).
+
+      - The *freeze tapscript* contains any authorization conditions
```

Comment:
Does it make sense to note here that the freeze doesn’t "require" a spend delay? (as I think this is implied when we note later in the writeup that Alice can spend using the freeze tapscript branch if she notices shenanigans)

Reply:
Add mention about time delays (or lack thereof) to preceding paragraph and also added several other time mentions throughout the text to help clarify this point.  Thanks!